recipes/gtsam/all/conanfile.py

```diff
@@ -160,6 +164,39 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "CMake"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
+
+        self.cpp_info.names["cmake_find_package"] = "GTSAM"
```

Comment:
I added support for components, but I have no idea what I'm doing TBH. Please review.
I kept the targets lowercase since `GTSAM-exports.cmake` has them declared that way
```
foreach(_expectedTarget CppUnitLite gtsam gtsam_unstable)
...
```